pywbem/mof_compiler.py

```diff
@@ -1627,7 +1633,10 @@ def p_instanceDeclaration(p):
             raise ce
 
     if alias:
+        if keybindings:
+            inst.path.keybindings = keybindings
```

Comment:
Actually the it always has the path set.

on line 1596 it first creates the path and then the new instance, putting the path it created into the new instance.

We are clearly wildely inconsistent here in that the only time this path is really correct is when the alias is set.  Otherwise it is only the name component of the path.

But the only change we are really making is to add the keybindings if the alias is set.